src/components/Eventyay/LeadScanning.vue

```diff
@@ -0,0 +1,14 @@
+<script setup>
+import QRCamera from '@/components/Common/QRCamera.vue'
+import { useLoadingStore } from '@/stores/loading'
+const loadingStore = useLoadingStore()
+loadingStore.contentLoaded()
+</script>
+
+<template>
+  <div
+    class="-mt-16 grid h-screen w-full grid-cols-1 place-items-center items-center justify-center align-middle"
+  >
+    <QRCamera :qr-type="'eventyaylead'" :scan-type="'Lead-Scan'" />
```

Comment:
Same as previous comment, when you are passing static value, just pass in the form of HTML attribute, it is clearer than VueJS synxtax.

Reply:
Done 👍